pkgs/development/python-modules/pympler/default.nix

```diff
@@ -13,14 +16,37 @@ buildPythonPackage rec {
     sha256 = "993f1a3599ca3f4fcd7160c7545ad06310c9e12f70174ae7ae8d4e25f6c5d3fa";
   };
 
+  patches = [
+    # Fixes a TypeError on Python 3.11
+    # (see https://github.com/pympler/pympler/issues/148)
+    (fetchpatch {
+      name = "${pname}-pull-149.patch";
```

Comment:
```suggestion
      name = "python-3.11-compat.patch";
```

Reply:
Hmm, okay. Is there a reason we don't prefix patch names with pname? Seems like the only purpose of giving them names is to be informative to anyone browsing the store; and without the context of the pname isn't `python-3.11-compat.patch` still pretty unhelpful?

Reply:
It's helpful in the build logs, when looking at a package's tree of build dependencies and it's helpful for some introspective things - e.g. security scanners can detect patches named `CVE-XXXX-XXXX.patch` and infer that it's a fix for that issue.

There's probably no harm in adding a `pname` but I've never felt the need - not many people go browsing the store randomly.